pandas/tests/io/parser/common/test_read_errors.py

```diff
@@ -161,10 +164,10 @@ def test_suppress_error_output(all_parsers):
     # see gh-15925
     parser = all_parsers
     data = "a\n1\n1,2,3\n4\n5,6,7"
-    expected = DataFrame({"a": [1, 4]})
+    expected = Series([1, 4], name="a", index=[0, 1])
```

Comment:
why change this?

Reply:
seems like there's no need to change this test. I rolled back my changes here.